include/cocaine/rpc/dispatch.hpp

```diff
@@ -285,6 +292,7 @@ struct slot_builder<Event, executor_state<F, R>> {
     ~slot_builder() noexcept(false) {
         typedef typename aux::select<
             R,
+            typename std::remove_reference<decltype(fn)>::type,
```

Comment:
because of
`/usr/local/include/cocaine/utility.hpp:82:47: error: type '(lambda at /Users/antmat/workspace/cocaine-plugins/logging/src/logging_v2.cpp:526:15) &' cannot be used prior to '::'
      because it has no members
struct result_of : public result_of<decltype(&F::operator())> {};`